utils/src/jsMain/kotlin/com/badoo/reaktive/utils/Timers.kt

```diff
@@ -0,0 +1,42 @@
+package com.badoo.reaktive.utils
+
+import kotlin.browser.window
+
+external fun setTimeout(handler: dynamic, timeout: Int = definedExternally, vararg arguments: Any?): Int
```

Comment:
Please make them `internal`

Reply:
Why do we need `definedExternally` here?

Reply:
just wanted to match the original contract. Will remove and test if it works, then I will remove it